Bitrise/testing_bitrise.yml

```diff
@@ -21,9 +31,10 @@ workflows:
             fi
 
             # Get the latest master branch for WeTransfer-iOS-CI if the submodule exists
-            git submodule update --remote --no-fetch Submodules/WeTransfer-iOS-CI
+            # git submodule update --remote --no-fetch Submodules/WeTransfer-iOS-CI
```

Comment:
Is there a reason this is commented out rather than removed?

Reply:
Good question

Reply:
Ah, yeah, there is! I should've explained that.

It's commented out to allow testing this branch specific changes inside Mule and our SDK since it prevents fetching latest from master.

I'm changing it right before merging!